modules/@ergonode/media/src/components/Forms/ResourceRelationsForm.vue

```diff
@@ -19,7 +19,7 @@
                                     v-for="relation in row.relations"
                                     :key="relation.name"
                                     :title="relation.name"
-                                    :link="relation._link"
+                                    :link="relation._link || {}"
```

Comment:
By setting default prop value as new object to `link` it will not be necessary to check it at this lvl

Reply:
This is necessary at this level because the backend cuts the `_link` property if it does not have permission to the resource

Reply:
Component `LinkRelationButton` has a property of the object type, so in order not to give different types it passes the empty object